### DISCUSSION TITLE
SOPs not recognizing Azure CLI location

### DISCUSSION THREAD

**Discussion by jronnomo** [getsops/sops#1378] [Q&A]
Hi all,

I'm trying to execute sops v3.7.3 decryption on a secrets.yaml. This version works for others on my team - I'm the only one receiving this error. I have /c/Program Files/Microsoft SDKs/Azure/CLI2/wbin included in my PATH with az.cmd sitting within the directory. I've logged in using 'az login' and have confirmed I have access to the keyvault. I can list the key using 'az keyvault key list --vault-name kv-dcentriq-gitops-devqa'

However, when running 'sops --config '' -d ./secrets.yaml', I get the below error:

[AZKV]   time="2023-12-15T11:25:55-05:00" level=error msg="Failed to create Azure authorizer" error="Invoking Azure CLI failed with the following error: 'az' is not recognized as an internal or external command,\r\noperable program or batch file.\r\n"
Failed to get the data key required to decrypt the SOPS file.

Group 0: FAILED
  https://kv-dcentriq-gitops-devqa.vault.azure.net/keys/sops-devqa-cluster/xx: FAILED
    - | Invoking Azure CLI failed with the following error: 'az' is
      | not recognized as an internal or external command,
      | operable program or batch file.

Recovery failed because no master key was able to decrypt the file. In
order for SOPS to recover the file, at least one key has to be successful,
but none were.

Any idea what's causing the issue?

**Comment by Berndwl** [comment]
A coworker of mine also faced the same issue.
azure cli 2.51.0 introduced the option to download the 64 bit version for windows, and it looks like this causes the above. 

You could try downloading the 32-bit version and change your path to this version (and uninstall the 64-bit one)

**Comment by mikeblakeuk** [comment]
Is this the final answer?
Why can't sops use teh 64 version?
